pandas/core/arrays/timedeltas.py

```diff
@@ -21,6 +21,19 @@
 from pandas.tseries.frequencies import to_offset
 
```

Comment:
can you change to absolute imports, I think we are moving away from the relative ones fro readability.